charts/kubearchive/templates/role_binding.yaml

```
@@ -15,3 +15,16 @@ subjects:
     name: {{ $.Values.kubearchive.serviceAccount }}
     namespace: {{ .name }}
 {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Values.apiServer.clusterRoleBinding }}
```

Call everything the same so its easier to track.

```suggestion
   name: {{ .Values.apiServer.name }}
```